scripts/generate-encoded-passwords-module.js

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const path = require('path');
+const readPasswords = require('./lib/read-passwords');
+const { Encoder } = require('incremental-encoder').default;
+
+const inputFilename = process.argv[2];
+const numberOfPasswords = parseInt(process.argv[3]);
+const minLength = parseInt(process.argv[4]) || 8;
+
+if (! inputFilename || ! numberOfPasswords) {
+  console.error(`Usage: node ${path.basename(process.argv[1])} <input filename> <number of passwords> [<min length>]`); //eslint-disable-line no-console
+  return 1;
+}
+
+const passwords = readPasswords(inputFilename, minLength, numberOfPasswords);
+const encoder = new Encoder();
+const encodedPasswords = encoder.encode(passwords);
+
+/*eslint-disable no-console*/
+console.log('/* This Source Code Form is subject to the terms of the Mozilla Public');
+console.log(' * License, v. 2.0. If a copy of the MPL was not distributed with this');
+console.log(' * file, You can obtain one at http://mozilla.org/MPL/2.0/. */');
+console.log('');
+console.log(`module.exports = \`${encodedPasswords.join('\n')}\`;\n`);
```

Comment:
Was there a particular reason for console.log instead of writing to a file?  Just curious.

Reply:
The only reason I could think of is it's nice to be able to use unix tools on output, but that's not really a strong reason here. I can change it to write directly to a file instead.